### DISCUSSION TITLE
V3 feedback api

### DISCUSSION THREAD

**Discussion by seattlesuekim** [etsy/open-api#693] [Ideas]
I noticed that the v3 api doesn't include any of the Feedback methods (`findAllUserFeedbackAs...`). Is there any plan to include these in v3? A number of well-loved third party seller tools rely on this, and Etsy sellers all over the world will have diminished confidence without these tools.

**Comment by etsyachristensen** [comment]
At this point these are still on the [Retired Resources](https://developers.etsy.com/documentation/migration/retired-resources) list, but  I'll move this to the Idea category where we can track it as a request for new features in the V3 API.

**Reply by seattlesuekim** [reply]
Thank you! I have many users of my app reaching out to me asking about this, would it be helpful if I reported the number of requests I'm getting about this, to show the high demand for the feedback api?

**Reply by etsyachristensen** [reply]
Can't hurt! Might help. :-)

**Reply by seattlesuekim** [reply]
Thanks - I posted about this on Reddit (/r/EtsySellers), and as the result of a single post, received 157 results very quickly to a Google survey that I created.

The mean rating for usefulness of my tool was 9.07 out of 10. These responses represent $4.3 million in annual sales. If this is a representative sample of my users, this extrapolates to $356 million in annual sales. Some quotes from my survey: 

"I would probably stop using Etsy altogether if I didn’t have this service! As a small business It’s just too risky without it.”

"I rely on this service to run a successful shop. If we can check buyer ratings on Depop, we should also be able to do the same for Etsy.”

"Please don't take this tool away from us, it's saved me so many headaches from customers that Etsy themselves don't give us the tools to manage”

According to my Google Analytics, my tool has 5000+ active Chrome Extension users as well as around 13K active weekly users for my website. Both the extension and the website primarily use the Feedback API.

Beyond the survey, I’ve received dozens of heartfelt messages from artists and craftspeople saying that they would have given up on selling with Etsy long ago if not for my services, and that it dramatically changes their experience. The primary positive outcome is that people feel more confident in their sales, since scammers/bad customers are more rare than people might assume. This is a free service available to all, and I maintain these tools as a volunteer because I see the good that they do.

[Here is the raw data from the survey](https://docs.google.com/spreadsheets/d/1a5Pi8P63wqJJqqQXP_IJh3lOqdDuO7uiDUhTQ7tCuO4/edit?usp=sharing)

Appreciate this issue being tracked

**Comment by benjymouse** [comment]
+1 would be great if this makes it into v3. it helps us spot scammers much faster and we can sell with confidence

**Comment by dangrossman** [comment]
I use EtsyCheck and KarenCheck all the time. Checking the feedback left by customers is invaluable in identifying if you're dealing with a scammer after someone sends you a message as a seller. I would hate to see these services die because you've sunset the API they need to work.

**Reply by chrishilbert** [reply]
In fairness, if Etsy would just give us detailed buyer reporting/inspection, EtsyCheck/KarenCheck wouldn't be necessary.  For now they are.

**Reply by jonmlee** [reply]
+1 to this and all the other CRM tools they're missing.

**Comment by seattlesuekim** [comment]
Hello, any update on this? I need to let my users know whether my service will be shut down with enough notice for them to take any actions. Thank you!

**Comment by trungpv1601** [comment]
+1
